assets/css/custom-props/theme-dark.css

```
@@ -80,6 +80,7 @@ body.dark {
   --sidebarHover:                var(--white);
   --sidebarScrollbarThumb:       var(--coldGray);
   --sidebarScrollbarTrack:       var(--sidebarBackground);
+  --sidebarStaleVersion:         var(--orangeDark);
```

```suggestion
  --sidebarStaleVersion:         var(--orangeLight);
```

The brighter orange goes in the darker barkground. :)